app/Models/Question.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Question extends Model
+{
+    use HasFactory;
+    
+    protected $fillable = [
+       
+        'survey_id',
+        'question',
+        'description',
+        'question_type_id',
+    ];
+
+    /**
+     * Returns the survey which this question belongs to
+     * 
+     * @Return belongsTo
+     * 
+     */
+    public function survey()
+    {
+        return $this->belongsTo(Survey::class);
+    }
+
+    /**
+     * Returns the options associated with this question
+     * 
+     * @Return hasMany
+     * 
+     */
+    public function questionOptions()
```

Comment:
```suggestion
    public function options()
```
You already are in the question model so to prevent $quest->questionOptions i think $question->options is a lot cleaner.